rocketpy/mathutils/function.py

```diff
@@ -250,20 +248,25 @@ def set_source(self, source):
             # if Function is 1D, sort source by x. If 2D, set z
```

Comment:
This comment is a bit ambiguous. 

The correct sentence should be: "set x and y. If Function is 2D, also set z."

Reply:
Reading the code again...

If Function is 5D, for instance, the x and y array won't be set. Is that expected?

Reply:
I'll change it so it keeps consistency with old code. Done in https://github.com/RocketPy-Team/RocketPy/pull/582/commits/0bd6f6396e8716fb0c76025dd2e29380528cfa0d